experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -833,7 +833,7 @@ describe("FormAudience", () => {
     expect(screen.queryByTestId("languages")).not.toBeInTheDocument();
     expect(screen.queryByTestId("locales")).toBeInTheDocument();
   });
-  it("disables isFirstRun for desktop", async () => {
+  it("isFirstRun does not renders for mobile application", async () => {
```

Comment:
it should be desktop application

Reply:
@yashikakhurana sorry for the typo fixed it 👍🏻